submarine-server/server-core/src/test/java/org/apache/submarine/server/AbstractSubmarineServerTest.java

```diff
@@ -237,10 +263,12 @@ protected static PostMethod httpPost(String path, String request, String mediaTy
     PostMethod postMethod = new PostMethod(URL + path);
     postMethod.setRequestBody(request);
     postMethod.setRequestHeader("Content-type", mediaType);
-    postMethod.getParams().setCookiePolicy(CookiePolicy.IGNORE_COOKIES);
+    //postMethod.getParams().setCookiePolicy(CookiePolicy.IGNORE_COOKIES);
```

Comment:
Should we delete it?

Reply:
> Should we delete it?

Of course, with REST we should no longer explicitly support session/cookie later on.

Reply:
SGTM